quickwit-storage/src/bundle_storage.rs

```diff
@@ -111,11 +111,8 @@ impl BundleStorageFileOffsets {
     }
 
     /// Return file offsets for given path.
```

Comment:
```suggestion
    /// Returns file offsets for given path.
```

Reply:
Good point. I added a ticket to merge first, and improve later.